GitCommands/RevisionReader.cs

```diff
@@ -111,9 +113,8 @@ public sealed class RevisionReader : IDisposable
 
             var arguments = BuildArguments(maxCount, refFilterOptions, branchFilter, revisionFilter, pathFilter);
 
-#if TRACE
+#if DEBUG
```

Comment:
`TRACE` is available in release builds, and can be viewed in [DebugView](https://docs.microsoft.com/sysinternals/downloads/debugview)

Reply:
Yes, this was intentional. I do not see a need for this in released builds
Of course, performance tests need to manually enable this.
While this is likely the most important piece of code to optimize opening of repos, it is not the only part that need tweaking.
This could be controlled with a app setting too (probably just in the settings file, not in the settings form).

Reply:
Then we should replace just `Trace` calls with `Debub` calls :)

Reply:
will change this printout
(printouts for ParseAssert will remain)

Reply:
Added a comment that this should be possible to activate
#if DEBUG must remain